configure.ac

```diff
@@ -228,7 +228,8 @@ if test "$os_unix" = "yes"; then
 	                          [Whether vsock support available])
 	                AC_MSG_RESULT([yes])],
 	               [AC_MSG_RESULT([no])])
-
+else
+    AM_CONDITIONAL([NEED_READPASSPHRASE], [false])
```

Comment:
I would add some comment explaining why this can be set to `false` on Windows (i.e., passphrase query on terminal is only supported on Unix platforms).